### ISSUE TITLE
Can't set fps cap to 60 (Steam Deck)

### ISSUE THREAD

**Issue by Samuel-16** [anegostudios/VintageStory-Issues#7885]
### Game Version

v1.12.6 (Stable)

### Platform

Linux

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

Not sure whether I should put it here or in suggestions, but it would help to be able to select from common fps values or enter a desired fps as a number. It won't let me set it to 60, as my Steam Deck resolution seems to cause it to skip values.

I think being unable to select common fps values counts as a bug.

### How to reproduce

Steam Deck resolution is 1280x800, so you may be able to recreate this behaviour by setting the window size to that resolution. If not, try setting it as your monitor size or seeing if you can find someone who has access to a Steam Deck in your team.

### Screenshots

![Image](https://github.com/user-attachments/assets/fd0e4fdf-3706-4b1a-96a0-b28871bb7150)

### Logs (Required)

[client-debug.log](https://github.com/user-attachments/files/24890884/client-debug.log)
[client-main.log](https://github.com/user-attachments/files/24890885/client-main.log)

Not sure if setting the FPS was attempted in that session, but I think the logs _probably_ won't matter too much in this case.
